packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -1316,8 +1316,7 @@ void main() {
   });
 
   // Regression test for https://github.com/flutter/flutter/issues/154532.
-  testWidgets('Keyboard navigation does not throw when no entries match the filter',
-    (WidgetTester tester) async {
+  testWidgets('Keyboard navigation does not throw when no entries match the filter', (WidgetTester tester) async {
       await tester.pumpWidget(MaterialApp(
         home: Scaffold(
           body: DropdownMenu<TestMenu>(
```

Comment:
Looks great. Could we un-indent the function body as well?
```suggestion
  testWidgets('Keyboard navigation does not throw when no entries match the filter', (WidgetTester tester) async {
    await tester.pumpWidget(MaterialApp(
      home: Scaffold(
        body: DropdownMenu<TestMenu>(
```

Reply:
Got it. Thanks for the review and helping me fixing the format issues.

Reply:
No problem, thanks for these updates! I think we're good to merge now, assuming checks pass 👍